pkg/azure/actuator.go

```diff
@@ -108,6 +95,19 @@ func (a *Actuator) needsUpdate(ctx context.Context, cr *minterv1.CredentialsRequ
 	if !exists {
 		return true, nil
 	}
+
+	// Manual mode just update
+	credentialsMode, _, err := utils.GetOperatorConfiguration(a.client, logger)
+	if err != nil {
+		logger.WithError(err).Error("error loading CCO configuration to determine valid mode")
+		return true, err
+	}
+	if credentialsMode == operatorv1.CloudCredentialsModeManual {
+		return true, nil
```

Comment:
Ideally, we would actually check if the secret differs from the source of truth similar to AWS mint mode. However, this can be taken away as tech debt.